code/modules/ftl/machinery/ftl_drive.dm

```diff
@@ -50,6 +81,8 @@
 	if(!istype(get_area(src), /area/shuttle/ftl) || (SSstarmap.ftl_drive && isturf(SSstarmap.ftl_drive.loc)))
 		stat |= BROKEN
 		return
+	var/obj/item/weapon/circuitboard/machine/ftl_drive/B = new
```

Comment:
What will this accomplish? The circuitboard will be removed once Initialize() is over

Reply:
it applies the default parts to the machine, otherwise I think the machine would be there, but it wouldn't have the stock parts in it (the tier 1 research parts like manips, caps)